packages/polaris-viz/src/components/VerticalBarChart/components/VerticalBar/VerticalBar.tsx

```diff
@@ -9,9 +9,9 @@ import {
 import {ZeroValueLine} from '../../../shared/ZeroValueLine';
```

Comment:
This used to be called `Bar` but we also have a `Bar` component in `/shared`. 

Eventually we'll look into using a single component, but to make things less confusing in the short term, I'm renaming this one to `VerticalBar`.